extensions/barrage/src/main/java/io/deephaven/extensions/barrage/util/BarrageUtil.java

```diff
@@ -657,10 +654,10 @@ private static ArrowType arrowTypeFor(Class<?> type) {
                     return Types.MinorType.LIST.getType();
                 }
                 if (type == LocalDate.class) {
-                    return LOCAL_DATE_TYPE;
+                    return Types.MinorType.DATEDAY.getType();
```

Comment:
Do we want to support the other format? I'm OK with picking one for this PR, but we should support receiving both in the future.